hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/repository/PrisonerRepository.kt

```diff
@@ -39,6 +49,40 @@ class PrisonerRepository(
     openSearchRestTemplate.index(IndexQueryBuilder().withObject(prisoner).build(), index.toIndexCoordinates())
   }
 
+  fun getSummary(prisonerNumber: String, indices: List<SyncIndex>): PrisonerDocumentSummary? =
+    indices.firstNotNullOfOrNull { index ->
+      client.get(GetRequest(index.indexName, prisonerNumber), RequestOptions.DEFAULT)
+        .toPrisonerDocumentSummary(prisonerNumber)
+    }
```

Comment:
Using the RestHighLevelClient to get metadata (i.e. version info) as well as the document